### PR TITLE
Update rules for focal that are needed by rospilot package

### PR DESCRIPTION
**Pull request by cberner**
postgresql-postgis provides geospatial indexing with Postgresql
* Ubuntu: https://packages.ubuntu.com/focal/postgresql-12-postgis-3

python3-cherrypy3 provides CherryPy a minimalist web framework
* Ubuntu: https://packages.ubuntu.com/focal/python3-cherrypy3
* Debian: https://packages.debian.org/buster/python3-cherrypy3

python3-colorama is a library to output colored terminal text
* Ubuntu: https://packages.ubuntu.com/focal/python3-colorama
* Debian: https://packages.debian.org/buster/python3-colorama

python3-mapnik is a library for developing GIS mapping applications
* Ubuntu: https://packages.ubuntu.com/focal/python3-mapnik
* Debian: https://packages.debian.org/buster/python3-mapnik

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1002,7 +1002,9 @@ python-cherrypy:
   debian: [python-cherrypy3]
   fedora: [python-cherrypy]
   gentoo: [dev-python/cherrypy]
-  ubuntu: [python-cherrypy3]
+  ubuntu:
+    '*': [python-cherrypy3]
+    focal: [python3-cherrypy3]
```
Comment (clalancette) [inline rosdep/python.yaml:1007]: To support python3, please leave the existing `python-` keys (which are implicitly Python 2), and instead add new `python3-` keys.  That goes for all of the ones below as well.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (cberner) [reply rosdep/python.yaml]: Ah, sorry about that. I've updated it now

**Review by clalancette — APPROVED** [review]

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
@cberner the rules look good but could you please update the pull request description to follow our [contribution guidelines for rosdep keys](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules).

**Comment by cberner** [comment]
@nuclearsandwich ah, sorry about that. I've updated it. Is that correct?

**Comment by nuclearsandwich** [comment]
@cberner with the merge of #25617 it seems like this PR is completely replaced. If there's something I'm missing (or if I broke something in the merge conflict resolution please let me know and I'll correct it. Otherwise I think we can close this PR. Thanks!

**Comment by cberner** [comment]
Looks good, thanks!

**Pull request closed without merging**